adambushman/public-library-web-app
# Adam/events page

## Description

I created the necessary "view" pages for all events (list) and a single event (for more detail).

Also, I added a "schedule" page where we can see all the classes and events, with tool tips and links to the detail pages.


## How to test

Use git pull then git checkout adam/events-page. Launch the WAMP server and navigate to the "events" page. Start clicking through the content. Also check out the "schedule" page and experiment with hovers and clicks.
